src/components/molecules/ChatsList/ChatsList.tsx

```diff
@@ -45,28 +45,11 @@ const ChatsList: React.FunctionComponent = () => {
   const { user } = useUser();
   const dispatch = useDispatch();
 
-  const chatUsers = useSelector(chatUsersSelector) ?? {};
+  const chatUsers = useSelector(chatUsersByIdSelector) ?? {};
```

Comment:
Do/can we have a hook that wraps this up along with it's associated 'connect' call (similar to what we are doing in the users PR)

I'll only comment on this once in this review, but my request would be that all occurrences of it are updated in that same way, to use the hook.